template/android/settings.gradle

```diff
@@ -6,4 +6,6 @@ includeBuild('../node_modules/react-native-gradle-plugin')
 if (settings.hasProperty("newArchEnabled") && settings.newArchEnabled == "true") {
     include(":ReactAndroid")
     project(":ReactAndroid").projectDir = file('../node_modules/react-native/ReactAndroid')
+    include(":ReactAndroid:hermes-engine")
+    project(":ReactAndroid:hermes-engine").projectDir = file('../node_modules/react-native/ReactAndroid/hermes-engine')
```

Comment:
nice ^ spotted/fixed in the nightly too earlier

but lagging [few days behind](https://github.com/leotm/react-native-template-new-architecture/pull/554/commits/bbf0fa767a5716c12296ed9ce3a05d4e831fa827) 🤪 more contributions inc